.github/workflows/ci.yml

```diff
@@ -2,16 +2,18 @@ name: ci
 on:
   push:
     branches: ['*']
-  pull_request:
-    branches: ['*']
 jobs:
   validate:
+    strategy:
+      matrix:
+        scala: [2.13.3, 2.12.10]
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
       with:
         fetch-depth: 0
     - uses: olafurpg/setup-scala@v10
     - uses: coursier/cache-action@v5
-    - run: sbt validate
-    - uses: codecov/codecov-action@v1
+    - run: sbt ++${{ matrix.scala }} validate
+    - if: matrix.scala == '2.13.3'
+      run: sbt "++${{ matrix.scala }} docs/run"
```

Comment:
I'm not a fan of having scala versions here, and essentially overriding whatever is defined in `build.sbt`.